eng/release/Scripts/InstallAzCopy.ps1

```diff
@@ -0,0 +1,48 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)][string] $ToolsDirectory,
+    [Parameter(Mandatory=$false)][string] $TaskVariableName = $null
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2.0
+
+$url = 'https://aka.ms/downloadazcopy-v10-windows'
+$basePath = Join-Path $ToolsDirectory 'azcopy'
+
+$zipPath = Join-Path $basePath 'azcopy.zip'
+$toolDirPath = Join-Path $basePath 'azcopy'
+$azCopyPath = Join-Path $toolDirPath 'azcopy.exe'
+
+if (Test-Path $azCopyPath) {
+    Write-Verbose 'Already installed'
+} else {
+    if (!(Test-Path $basePath)) {
+        New-Item -ItemType 'Directory' -Path $basePath | Out-Null
+    }
+
+    Write-Verbose 'Fetching...'
+    Invoke-WebRequest -Uri $url -OutFile $zipPath
```

Comment:
I'm not sure this has been fixed, but Invoke-WebRequest is known to be pretty slow. There may be alternatives that are a bit faster. Not sure how big the file is.

Reply:
It's somewhat slow, takes a little less than 30 seconds for the whole task to execute.